utils_cv/tracking/data.py

```diff
@@ -0,0 +1,4 @@
+class Urls:
```

Comment:
Can you please have a look at e.g. object detection and copy how the class looks there including the all() function